### PR TITLE
Documentation:teaching:labs:device_drivers update

### PR DESCRIPTION
**Pull request by teoxyz95**
Fixed my_write(), was same as my_read().

### CODE REVIEW

File: Documentation/teaching/labs/device_drivers.rst [hunk 1]
```diff
@@ -561,17 +561,12 @@ The structure of the write function is similar:
...
-	   return 0;
 
        /* read data from device in my_data->buffer */
-       if (copy_to_user(user_buffer, my_data->buffer, len))
+       if (copy_from_user(my_data->buffer, user_buffer, my_data->size))
 	   return -EFAULT;
-
-       *offset += len;
-       return lent;
+
```
Comment (dbaluta) [inline Documentation/teaching/labs/device_drivers.rst]: Hi Robert,

This is a very good catch. Anyhow, I would like you to do some changes:

1) The commit message should be more clear.

You can use:

Documentation: teaching: labs: Fix data transfer

At write we need to get data from user and write them to device, so use copy_from_user.

2) Also update the comment, it should say /* write data from my_data->buffer to device.

Comment (teoxyz95) [reply Documentation/teaching/labs/device_drivers.rst]: Thank you for your speedy reply, I will adjust my commit.
From now on please use my name. 
My name is Teo not Robert

Comment (teoxyz95) [reply Documentation/teaching/labs/device_drivers.rst]: Also I used copy_from_user, I do not see where the problem lies.

Comment (dbaluta) [reply Documentation/teaching/labs/device_drivers.rst]: Hi Teo,

Sorry! I was replying to two pull requests in the same time :)

**Comment by razvand** [comment]
@teoxyz95 , this is an old pull request and can't be easily merged. @valx90 , take a look in it, then create a new pull request (if the problem still exists) and close this one.

**Comment by valx90** [comment]
A fix for this issue was already merged in https://github.com/linux-kernel-labs/linux/commit/6c55b37e87fb902a22bb343ceea9e87f163a3b95#diff-fbe578887b891e0774e6b3ca14821866. I will close this PR.

**Pull request closed without merging**